website/docusaurus.config.js

```diff
@@ -301,7 +301,22 @@ module.exports = {
         },
       },
     ],
-    ['docusaurus-plugin-sass', {}]
+    ['docusaurus-plugin-sass', {}],
+    [
+      '@docusaurus/plugin-client-redirects',
+      {
+        redirects: [
+          {
+            from: ['/docs/general/community', '/docs/general/subscribe-guide'],
+            to: '/docs/general/join',
+          },
+          {
+            from: '/docs/general/submit-issue',
+            to: '/docs/general/contributor-guide',
+          },
```

Comment:
We should not need this right?

Reply:
![image](https://user-images.githubusercontent.com/48400568/160751625-e38641dc-dd34-46df-a581-822f87c0f897.png)

It's from here.

Reply:
I found the problem.

Reply:
I fixed this.